Libraries/Components/WebView/WebView.android.js

```diff
@@ -127,13 +134,25 @@ var WebView = React.createClass({
       domStorageEnabled = this.props.domStorageEnabledAndroid;
     }
 
+    var htmlProp, urlProp, sourceProp;
+    if(this.props.html) {
+      console.warn('html is deprecated. Use source instead.');
+      htmlProp = this.props.html;
+    } else if(this.props.url) {
+      console.warn('url is deprecated. Use source instead.');
```

Comment:
This is a good idea, do you have a recommendation for where this deprecatedPropType method should go so that any component can access it?